watertap/property_models/tests/test_multicomp_aq_sol_prop_pack.py

```diff
@@ -238,7 +238,9 @@ def model2():
 
     m.fs = FlowsheetBlock(dynamic=False)
     m.fs.properties = MCASParameterBlock(
-        solute_list=["A", "B", "C", "D"], charge={"A": 1, "B": -2, "C": 2, "D": -1}
+        solute_list=["A", "B", "C", "D"], 
+        charge={"A": 1, "B": -2, "C": 2, "D": -1},
+        mw_data={"A": 10e-3, "B": 25e-3, "C": 100e-3, "D": 25e-3}
```

Comment:
`solute_list`: if omitted, a `ConfigurationError` is raised.
`charge`: if omitted, a warning will be logged.
`mw_data`: if omitted, a `ConfigurationError` is raised.